migrations/20230620194206_webxdc_versions_and_alter_config.sql

```diff
@@ -0,0 +1,11 @@
+-- Add migration script here
```

Comment:
This migration could be part of the initial migration, because atm. we not yet have to keep continuous db in mind as we can just always start from scratch.